docs/data/introduction/support/support.md

```diff
@@ -34,18 +43,25 @@ Including your Order ID (or Support key) in the issue helps us prioritize the is
 2. MUI X Premium: The same as MUI X Pro, but with higher priority.
 3. MUI X Priority Support add-on (coming soon): Provides a 24h SLA for the first answer.
 
-### Long-term support (LTS)
+## Long-term support (LTS)
 
-Bug fixes, performance enhancements, and other improvements are delivered in new releases. However, MUI remains committed to providing security updates and addressing regressions for the immediate predecessor of the current major version.
+Bug fixes, performance enhancements, and other improvements are delivered in new releases. However, we remain committed to providing security updates and addressing regressions for the immediate predecessor of the current major version.
```

Comment:
My thought was to try to fade the company name (MUI) from the docs. My idea is that we probably don't want to have it referenced at all. Go to https://react.dev/, you won't find the docs using "Facebook", but "We". Same thought on https://nextjs.org/docs. The value I see is that:

- Keep the audience focused on learning MUI X
- As MUI grows, it feels weird to reference an organization that is bigger than the project, as a reader of the docs, I would want to hear more the team's voice.